deploy/rbac-permissions-operator-config/05-osd-readers.ClusterRole.yaml

```diff
@@ -87,3 +66,281 @@ rules:
   verbs:
   - get
   - list
+### END
+### Start - Allow read permissions to more resources
+# https://issues.redhat.com/browse/OSD-5537
+- apiGroups:
+  - admissionregistration.k8s.io
+  resources:
+  - '*'
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - apiextensions.k8s.io
+  resources:
+  - customresourcedefinitions
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - apiregistration.k8s.io
+  resources:
+  - apiservices
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - authentication.k8s.io
+  resources:
+  - tokenreviews
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - authorization.k8s.io
+  resources:
+  - selfsubjectaccessreviews
+  - subjectaccessreviews
+  - selfsubjectrulesreviews
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - authorization.openshift.io
+  resources:
+  - clusterrolebindings
+  - subjectaccessreviews
+  - resourceaccessreviews
```

Comment:
Same as concern for other review resources, do we know why they're excluded from 'view' and what giving read access grants?

Reply:
Will remove it.

Reply:
done